docs(js): Add note regarding browser devtools not triggering errors (#3375)
getsentry/sentry-docs: src/platforms/common/index.mdx

<Note>

> Keep in mind that errors triggered within Browser DevTools are sandboxed and they will not trigger an error handler. Place the snippet directly in your code instead.

Reviewer: This will appear on the Getting Stated page for every SDK - which I suspect we don't want as this is a note specific to JS. 

I'd recommend instead putting this in the `includes/getting-started-verify/javascript.mdx` file.

```suggestion
Errors triggered from within Browser DevTools are sandboxed, so will not trigger an error handler. Place the snippet directly in your code instead.
```
